foyer/forcefield.py

```diff
@@ -210,6 +210,34 @@ def _update_atomtypes(unatomtyped_topology, res_name, prototype):
             for old_atom, new_atom_id in zip([atom for atom in res.atoms()], [atom.id for atom in prototype.atoms()]):
                 old_atom.id = new_atom_id
 
+def _separate_urey_bradleys(system, topology):
+        """ Separate urey bradley bonds from harmonic bonds in OpenMM System
+
+        Parameters
+        ---------
+        topology : openmm.app.Topology
+            Molecular structure to find atom types of
+        system : openmm System
+
+        """
+        atoms = [a for a in topology.atoms()]
+        bonds = [b for b in topology.bonds()]
+        ub_force = mm.HarmonicBondForce()
+        harmonic_bond_force = mm.HarmonicBondForce()
+        for i, force in enumerate(system.getForces()):
+            if isinstance(force, mm.HarmonicBondForce):
+                for j in range(force.getNumBonds()):
+                    if (atoms[force.getBondParameters(j)[0]], 
+                        atoms[force.getBondParameters(j)[1]]) not in bonds:
```

Comment:
Is the ordering of atoms within a bond guaranteed to be the same in both objects?